packages/xo-web/src/xo-app/dashboard/overview/index.js

```diff
@@ -141,10 +147,28 @@ class DefaultCard extends Component {
     })
   }
 
+  _canSendTheReport = createSelector(
+    () => this.props.plugins,
+    plugins => {
+      let count = 0
+      for (const key in plugins) {
+        const { id, loaded } = plugins[key]
```

Comment:
`for (const { id, loaded } of plugins) {`

Reply:
Plain objects are not iterable.